src/modules/Elsa.Studio.Workflows/Components/WorkflowInstanceViewer/Components/ActivityExecutionsTab.razor.cs

```diff
@@ -67,11 +71,12 @@ private void CreateSelectedItemDataModels(ActivityExecutionRecord? record)
         SelectedOutputData = outputData;
     }
 
-    private Task OnActivityExecutionClicked(TableRowClickEventArgs<ActivityExecutionRecordTableRow> arg)
+    private async Task OnActivityExecutionClicked(TableRowClickEventArgs<ActivityExecutionRecordTableRow> arg)
     {
-        SelectedItem = arg.Item.ActivityExecution;
+        var id = arg.Item.ActivityExecution.Id;
+        var fullRecord = await ActivityExecutionService.GetAsync(id);
```

Comment:
Can't we assign this directly to SelectedItem?

Reply:
Yes